### ISSUE TITLE
Discord bot

### ISSUE THREAD

**Issue by avaer** [webaverse/xrpackage#48]
Since the community is kind of rallying around the Webaverse Discord, it would be cool to polish up the Discord bot so we can share XRPK's and rooms with commands.

This is currently implemented as a Cloudflare worker that listens on Discord channels, with an AWS Lambda backend attached that handles Discord API calls.

If there is interest in this issue we can break it out to open source repos under `webaverse` and figure out a way to automate deployment.

**Comment by ChrisEddy** [comment]
I made pro bot for exokit before

**Comment by avaer** [comment]
Yup @ChrisEddy you have some experience here.

**Comment by ChrisEddy** [comment]
Where is that crypto code ?

**Comment by avaer** [comment]
There is a Cloudflare worker and AWS lambda functions, but it isn't crypto code -- it is more like Discord API code.

I can post those up if interested, though the deployment is quite bespoke so it will be hard to test.

**Comment by avaer** [comment]
Discord Cloudflare worker: https://github.com/webaverse/discord-cloudflare
Discord AWS Lambda function: https://github.com/webaverse/discord-lambda

**Comment by yahengsu** [comment]
Hey @avaer @ChrisEddy, I was wondering if any progress been made on this task?

**Comment by avaer** [comment]
No, nothing since the issue was posted. It's still something we'd like to do.

**Comment by yahengsu** [comment]
Sounds good! Are there any other features you think would be cool to see, apart from sharing xrpks/rooms with commands?

**Comment by avaer** [comment]
Yes, we already have the ability to list the packages in a certain user's blockchain wallet for example; this predates the web registry.

But ideally most of the functions of the `inspect.html` and `edit.html` frontend can be done from Discord commands.

- Create world
- Show world's packages
- Edit world's packages
- Set avatar
- Show inventory
- Add/remove package from inventory
- Export world scene
- Upload package from Discord file to inventory
- Show package contents/files
- Share screenshot of world

**Comment by avaer** [comment]
Ultimately I think it also makes sense to bridge in-world chat to discord as well, similar to how it's done with Hubs.

**Comment by shu8** [comment]
Just for reference, @yahengsu @nicoledanuwidjaja and I have made a repo for an initial discord bot at https://github.com/MLH-Fellowship/webaverse-bot!

**Comment by avaer** [comment]
Thanks, @ChrisEddy might be deploying it soon.
